kubernetes/README.md

```diff
@@ -44,22 +46,36 @@ eksctl create cluster \
     --nodes=2
 ```
 
-Install Nvidia device plugin to enable GPU support on your cluster.
+##### Create Microsot AKS
```

Comment:
typo: `Create Microsot AKS` -> `Microsoft AKS`

Reply:
Thank you very much. fixed.